### PR TITLE
change default foder and fix leafhub rejoin issue

### PR DESCRIPTION
**Pull request by ldpliu**
https://issues.redhat.com/browse/ACM-7881

### CODE REVIEW

File: operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-adhoc-investigation.yaml [hunk 1]
```diff
@@ -975,8 +1012,8 @@ data:
           {
             "current": {
               "selected": false,
-              "text": "",
-              "value": ""
+              "text": "por",
```
Comment (clyang82) [inline operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-adhoc-investigation.yaml:1015]: what does `por` mean?

Comment (ldpliu) [reply operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-adhoc-investigation.yaml]: > what does `por` mean?

Sorry, it's a mistake, and it's ok now

**Review by clyang82 — APPROVED** [review]

**Pull request merged**